include/fmt/core.h

```diff
@@ -138,6 +138,24 @@
 #  endif
 #endif
 
+// Check if typeid is available.
+#ifndef FMT_USE_TYPEID
+// In MSVC, typeid() is available with or without RTTI.
+#  if defined(_MSC_VER)
+#    define FMT_USE_TYPEID 1
+#  elif defined(__RTTI)  // EDG compilers.
+#    define FMT_USE_TYPEID 1
+#  elif defined(__INTEL_RTTI__)  // Intel compiler.
+#    define FMT_USE_TYPEID 1
+#  elif defined(__GXX_RTTI)  // G++.
+#    define FMT_USE_TYPEID 1
+#  elif FMT_HAS_FEATURE(cxx_rtti)  // Clang.
+#    define FMT_USE_TYPEID 1
+#  else
+#    define FMT_USE_TYPEID 0
+#  endif
+#endif
```

Comment:
Please move this to fmt/std.h where it is used.

Reply:
Done.